src/java.base/share/classes/java/time/format/DateTimeFormatterBuilder.java

```diff
@@ -1480,6 +1479,9 @@ public DateTimeFormatterBuilder appendLiteral(String literal) {
      * and no {@code HOUR_OF_DAY} is resolved, the parsed day period takes precedence.
      * If any conflict occurs in {@link ResolverStyle#LENIENT LENIENT} mode, no
      * exception is thrown and the day period is ignored.
+     * <p>
+     * "midnight" type allows both "00:00" as the start-of-day and "24:00" as the
```

Comment:
Editorial:  Add "The " at the beginning of the sentence.

Reply:
Fixed.